### ISSUE TITLE
Detach tab

### ISSUE THREAD

**Issue by jpfarias** [microsoft/terminal#12292]
# Description of the new feature/enhancement

I would like to be able to detach a tab into a new window. For instance, when I am working with multiple tabs and then I want
to watch 2 separate processes logs that are running on different tabs side by side.

Perhaps a good way to go about this is to add another entry to the context menu when the user right clicks the tab title, right after the  "Duplicate tab" option.

**Comment by j4james** [comment]
This sounds like a duplicate of #1256.

**Comment by jpfarias** [comment]
> This sounds like a duplicate of #1256.

I tried searching the open issues to check for duplicates before I posted this issue and I didn't find anything that mentioning detaching tabs. Perhaps the wording on #1256 tricked me since it says "tearoff" instead of detach.

Anyway, in this issue I also point to a change - or rather addition, to the context menu for right clicking the tab title that would let us detach the tab. 

I am sorry that I don't know about the inner workings of WT so I don't know how much work this would require.

Hopefully this isn't a big deal.

Thanks again for this great piece of software!
